From 44168d57fed931bf66cc923fbbdf9cca4bf39997 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:05:06 -0400
Subject: [redhat] kabi: Adding symbol pv_cpu_ops (arch/x86/kernel/paravirt.c)

Message-id: <16c2e2d608c7b2e15239050b14ac6c49d9f7edbf.1412771859.git.poros@redhat.com>
Patchwork-id: 97197
O-Subject: [RHEL7.1 PATCH 46/67] kabi: Adding symbol pv_cpu_ops (arch/x86/kernel/paravirt.c)
Bugzilla: 1148030 1148043 1148052
RH-Acked-by: Andrew Jones <drjones@redhat.com>

Bugzilla: 1148030,1148043,1148052
    https://bugzilla.redhat.com/show_bug.cgi?id=1148030
    https://bugzilla.redhat.com/show_bug.cgi?id=1148043
    https://bugzilla.redhat.com/show_bug.cgi?id=1148052

    Description:
    ============
    These symbols are required so that SGI modules play
    nice with paravirt in the kernel; yes we are trying
    to make guest OS more Numa aware.
    Symbol used in frs.ko, gru.ko, kbar.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pv_cpu_ops

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pv_cpu_ops b/redhat/kabi/kabi-module/kabi_x86_64/pv_cpu_ops
new file mode 100644
index 0000000..cb58df1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pv_cpu_ops
@@ -0,0 +1,2 @@
+#1-
+0x910538ff pv_cpu_ops vmlinux EXPORT_SYMBOL
-- 
1.7.1